airbnb_project/database/Dockerfile

```diff
@@ -0,0 +1,14 @@
+# Use the official Docker Hub Postgres image
+FROM postgres
+
+# Set the working directory
+WORKDIR /var/lib/postgresql/data
+
+# Set the required environment variable
+ENV POSTGRES_PASSWORD=@irBnbRegulation
```

Comment:
probably should not expose env setting here

Reply:
Agreed. This should be probably done on the server level and not here.